src/bin/websurfx.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! This module contains the main function which handles the logging of the application to the
 //! stdout and handles the command line arguments provided and launches the `websurfx` server.
-
+#[allow(unused_imports)]
```

Comment:
What's this for? You should use `#[cfg(feature = ...)]` for imports only used in features